packages/core/strapi/src/ee/license.ts

```diff
@@ -21,6 +21,7 @@ const DEFAULT_FEATURES = {
     // The default of 90 days is set in the audit logs service
     { name: 'audit-logs', options: { retentionDays: null } },
     { name: 'review-workflows' },
+    { name: 'cms-content-releases' },
```

Comment:
IIRC this name needs to correspond with the ChargeBee...? @Marc-Roig knows more than I, secondly, does it need `cms` prefixed? aren't all features for the cms? 😅

Reply:
Hey @Marc-Roig @joshuaellis this was a suggestion from cloud team considering they want to separate cms features from cloud features.